airflow/migrations/versions/0142_2_10_0_update_missing_constraints.py

```diff
@@ -0,0 +1,102 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Update missing constraints
+
+Revision ID: 686269002441
+Revises: 677fdbb7fc54
+Create Date: 2024-04-15 14:19:49.913797
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import literal
+
+# revision identifiers, used by Alembic.
+revision = '686269002441'
+down_revision = '677fdbb7fc54'
+branch_labels = None
+depends_on = None
+airflow_version = '2.10.0'
+
+
+def upgrade():
+    """Apply Update missing constraints"""
+    with op.batch_alter_table('connection') as batch_op:
+        batch_op.drop_constraint('unique_conn_id', type_='unique')
```

Comment:
Won't this fail for recently "freshly created from the ORM" dbs though? Should this whole block be under a "if `unique_conn_id` exists" condition?

Reply:
You're right cause if it's created with ORM and downgraded to say 2.8.4, upgrading again to the latest would mean that it won't exist